tests/e2e/tests/pilot/routing_test.go

```diff
@@ -58,30 +58,32 @@ func TestRoutes(t *testing.T) {
 	}()
 
 	cases := []struct {
-		testName      string
-		description   string
-		config        string
-		scheme        string
-		src           string
-		dst           string
-		headerKey     string
-		headerVal     string
-		expectedCount map[string]int
-		operation     string
-		onFailure     func()
+		testName        string
+		description     string
+		config          string
+		scheme          string
+		src             string
+		dst             string
+		headerKey       string
+		headerVal       string
+		expectedCount   map[string]int
+		operation       string
+		operationAlpha3 string
+		onFailure       func()
 	}{
 		{
 			// First test default routing
-			testName:      "a->c[v1=100]",
-			description:   "routing all traffic to c-v1",
-			config:        "rule-default-route.yaml",
-			scheme:        "http",
-			src:           "a",
-			dst:           "c",
-			headerKey:     "",
-			headerVal:     "",
-			expectedCount: map[string]int{"v1": 100, "v2": 0},
-			operation:     "default-route",
+			testName:        "a->c[v1=100]",
+			description:     "routing all traffic to c-v1",
+			config:          "rule-default-route.yaml",
+			scheme:          "http",
+			src:             "a",
+			dst:             "c",
+			headerKey:       "",
+			headerVal:       "",
+			expectedCount:   map[string]int{"v1": 100, "v2": 0},
+			operation:       "default-route",
+			operationAlpha3: "c.istio-system.svc.cluster.local:80/*",
```

Comment:
Please don’t do this. No more backward compatibility with old tests. If you are replacing something change everywhere uniformly. Otherwise these things linger forever and people forget to get rid of old ones.

Reply:
Removed. Ported the same change to v1.